client/state/reader/lists/reducer.js

```diff
@@ -72,7 +72,7 @@ export const subscribedLists = withSchemaValidation(
 			case READER_LISTS_RECEIVE:
 				return map( action.lists, 'ID' );
 			case READER_LISTS_FOLLOW_SUCCESS:
-				const newListId = get( action, [ 'data', 'list', 'ID' ] );
+				const newListId = action?.data?.list?.ID;
```

Comment:
It's not possible that `action` will be `null` / `undefined` here - if we get to this `switch` case, `action.type` will be equal to `READER_LISTS_FOLLOW_SUCCESS `.